NOTIF-91 Fix remove message for integrations

josejulio opened the pull request:
![Screenshot_2021-01-28_10-04-27](https://user-images.githubusercontent.com/3845764/106167549-ec069900-6152-11eb-8305-c7097e4e2e99.png)
![Screenshot_2021-01-28_10-04-10](https://user-images.githubusercontent.com/3845764/106167550-ec9f2f80-6152-11eb-91cf-e345ac204f61.png)


Depends on https://github.com/RedHatInsights/notifications-backend/pull/103

adamruzicka commented on src/PresentationalComponents/CustomInputFieldTemplate/CustomInputFieldTemplate.scss:
```diff
@@ -0,0 +1,5 @@
+@media (min-width: 768px) {
```
Patternfly uses `@screen-md-min` (and somewhere `$screen-md-min`), but I haven't found where these are defined

bastilian replied:
I think we can live with this. :)

bastilian commented:
Thanks @adamruzicka!

josejulio committed:
NOTIF-91 Fix remove message for integrations

josejulio committed:
fix plural

josejulio committed:
From UX review
 - Different message when there are no notifications
 - Includes the name in the message
